libraries/provision/ansible/playbooks/start-sg-accel.yml

```diff
@@ -24,7 +24,7 @@
 
   vars:
     sync_gateway_config_filepath:
-    couchbase_server_primary_node: "{{ hostvars[groups.couchbase_servers[0]].ansible_host }}"
+    couchbase_server_primary_node: ""
```

Comment:
Just being picky, some files have couchbase_server_primary_node: "" and some files have couchbase_server_primary_node: , without quotes. For consistency, can you add/remove everywhere.